src/redact/index.ts

```diff
@@ -21,3 +24,46 @@ export const addDefaultRedactPathStrings = (
   }
   return redact;
```

Comment:
@72636c , @etaoins : Seems like the default redact paths aren't added when `redact` is an object.

I feel like this may be an issue separate to this PR.

Reply:
I've fixed this in [Feat: Enable configuring default remove paths](https://github.com/seek-oss/logger/pull/84/commits/add2851dad173c2755f503043c6a1c1f2b8a2c54)